14 AsyncCalls_Redux_Saga/src/common/actions/httpCallStarted.ts

```diff
@@ -0,0 +1,8 @@
+const httpCallStarted = () => {
```

Comment:
Same here we can get rid of this, react promise tracker takes care of it

Reply:
also deleted :)